src/client/dfuse/ops/lookup.c

```diff
@@ -118,11 +119,12 @@ dfuse_reply_entry(struct dfuse_info *dfuse_info, struct dfuse_inode_entry *ie,
 
 			/* Save the old name so that we can invalidate it in later */
 			wipe_parent = inode->ie_parent;
-			strncpy(wipe_name, inode->ie_name, NAME_MAX);
+			memcpy(wipe_name, inode->ie_name, NAME_MAX);
```

Comment:
.

Reply:
- [ ] Restore, add comments and pragma on `strncpy()` usage

Reply:
> * [x]  Restore, add comments and pragma on `strncpy()` usage

Fixed with commit f38fd44a0